### PR TITLE
feat(ng-add): add x-prompts to install schematics

### PR DESCRIPTION
**Pull request by DevVersion**
* Recently the Angular CLI project introduced prompt support for schematics. Even though they seem to not run when using `7.0.0-beta.2` of the CLI, we should already ship the simple prompts for the `ng add` command.
* Renames the `/schematics/install` folder to `/schematics/ng-add`. (this makes it consistent with the actual schematic name and other packages that ship the `ng-add` schematic (e.g. `@angular/elements`)

### CODE REVIEW

File: src/lib/schematics/ng-add/schema.json [hunk 1]
```diff
@@ -19,12 +19,14 @@
     "theme": {
       "enum": ["indigo-pink", "deeppurple-amber", "pink-bluegrey", "purple-green", "custom"],
       "default": "indigo-pink",
-      "description": "The theme to apply"
+      "description": "The theme to apply",
+      "x-prompt": "What theme would you like to generate?"
```
Comment (jelbourn) [inline src/lib/schematics/ng-add/schema.json]: How about 
```
Enter a prebuilt theme name, or "custom" for a custom theme.
```

File: src/lib/schematics/ng-add/schema.json [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
     },
     "gestures": {
       "type": "boolean",
       "default": true,
-      "description": "Whether gesture support should be set up or not."
+      "description": "Whether gesture support should be set up or not.",
+      "x-prompt": "Would you like to set up gesture support?"
```
Comment (jelbourn) [inline src/lib/schematics/ng-add/schema.json]: How about
```
Add HammerJs for gesture recognition?
```

File: src/lib/schematics/ng-add/schema.json (hunk 1 above)
Comment (DevVersion) [reply src/lib/schematics/ng-add/schema.json]: That sounds better. Updated.

File: src/lib/schematics/ng-add/schema.json (hunk 2 above)
Comment (DevVersion) [reply src/lib/schematics/ng-add/schema.json]: Done

**Comment by DevVersion** [comment]
Not sure. There is nothing really in this PR that affects the @nodejs//:npm execution.

**Comment by jelbourn** [comment]
Just repush? (rerunning directly on circle ci is... problematic)

**Comment by DevVersion** [comment]
@jelbourn Yeah fixed now. I was mobile yesterday and couldn't repush so I just tried rerunning.

**Comment by jelbourn** [comment]
@DevVersion just needs rebase

**Comment by DevVersion** [comment]
@jelbourn Done.

**Pull request merged**